frontend/src/pages/add-blog.tsx

```diff
@@ -89,95 +89,99 @@ function AddBlog() {
+            />
+          </div>
 
-        <div className="mb-4 flex items-center flex-col md:flex-row ">
-          <label className="block font-semibold mb-2 w-full md:w-fit  md:mr-8">Categories</label>
-          <div className="flex flex-wrap gap-2">
-            {CATEGORIES.map((category) => (
-              <span
-                key={category}
-                className={`cursor-pointer
+          <div className="mb-4 flex items-center flex-col md:flex-row ">
+            <label className="block font-semibold mb-2 w-full md:w-fit  md:mr-8">Categories</label>
+            <div className="flex flex-wrap gap-2 dark:text-black">
+              {CATEGORIES.map((category) => (
+                <span
+                  key={category}
+                  className={`cursor-pointer
 									${
                     formData.categories.includes(category)
                       ? categoryProps(category, true)
                       : categoryProps(category, false)
                   }`}
-                onClick={() => handleCategoryClick(category)}
-              >
-                {category}
-              </span>
-            ))}
+                  onClick={() => handleCategoryClick(category)}
+                >
+                  {category}
+                </span>
+              ))}
+            </div>
           </div>
-        </div>
 
-        <div className="mb-4 flex items-center">
-          <label className="flex items-center">
-            <span className="text-gray-800 text-base font-medium">Featured Post</span>
-            <input
-              type="checkbox"
-              name="isFeaturedPost"
-              className="ml-2 h-5 w-5 text-indigo-600 border rounded focus:ring-indigo-400"
-              checked={formData.isFeaturedPost}
-              onChange={handleCheckboxChange}
-            />
-          </label>
-        </div>
+          <div className="mb-4 flex items-center">
+            <label className="flex items-center">
+              <span className="text-gray-800 text-base font-medium dark:text-white">
+                Featured Post
+              </span>
+              <input
+                type="checkbox"
+                name="isFeaturedPost"
+                className="ml-2 h-5 w-5 text-indigo-600 border rounded focus:ring-indigo-400  "
+                checked={formData.isFeaturedPost}
+                onChange={handleCheckboxChange}
+              />
+            </label>
+          </div>
 
-        <button
-          type="submit"
-          className="bg-black text-white flex items-center justify-center text-base p-2 rounded-lg hover:bg-gray-800 w-full md:w-fit"
-        >
-          Create Blog
-        </button>
-      </form>
+          <button
+            type="submit"
+            className=" dark:border-2 dark:border-white dark:bg-orange-400 bg-black text-white flex items-center justify-center text-base md:text-lg lg:text-xl p-2 rounded-lg hover:bg-gray-800 w-full md:w-fit"
+          >
```

Comment:
Lets have text-base for all the scenario, it is the latest changes done by me, please revert it back

Reply:
sir text-base is looking too small for laptop mode

Reply:
No, 16px is fine for even larger screen, if you make the text larger, you would have to also have a responsive padding for it , else it won't follow the UI guidelines

Reply:
Sir where can i find ui guidelines

Reply:
material 3 guidelines from google